data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/util/KafkaSourceSecurityConfigurer.java

```diff
@@ -236,10 +239,11 @@ public static String getBootStrapServersForMsk(final AwsIamAuthConfig awsIamAuth
         }
     }
 
-    public static void setAuthProperties(Properties properties, final KafkaSourceConfig sourceConfig) {
+    public static void setAuthProperties(Properties properties, final KafkaSourceConfig sourceConfig, final Logger LOG) {
```

Comment:
this class cannot have its own logger.?

Reply:
It is just a static class. But I am OK to put a different Logger for this class.